core/src/main/scala/Utils.scala

```diff
@@ -0,0 +1,17 @@
+package remotely
+
+import remotely.codecs.DecodingFailure
+import scodec.Err
+
+import scalaz.{\/-, -\/, \/}
+import scalaz.concurrent.Task
+
+package object utils {
+  implicit class AugmentedEither[E,A](a: E \/ A) {
+    def toTask(implicit conv: E => Throwable): Task[A] = a match {
+      case -\/(e) => Task.fail(conv(e))
+      case \/-(a) => Task.now(a)
+    }
+  }
+  implicit def errToE(err: Err) = new DecodingFailure(err)
```

Comment:
Are we sure we want to have implicit conversions to a throwable subclass floating around? That sounds error-prone to me.

If we are going to have it, it should at least be explicitly typed.